src/server/engine_shard_set.cc

```diff
@@ -868,6 +870,8 @@ ShardId Shard(string_view v, ShardId shard_num) {
       has_hashtags = true;
       v = v_hash_tag;
     }
+  } else if (absl::GetFlag(FLAGS_enable_round_robin_sharding)) {
+    has_hashtags = true;
```

Comment:
you dont use hash tags in sidekiq que name right?

Reply:
that's right. If this flag is enabled, we directly match it with `absl::StartsWith(key, round_robin_prefix_)` in `TryGetShardId`